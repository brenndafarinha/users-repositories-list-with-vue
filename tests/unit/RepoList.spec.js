import { shallowMount } from '@vue/test-utils';
import RepoList from '../../src/components/RepoList';
import store from '../../src/store'

describe('RepoList', () => {
    it("renders correctly", () => {
        const wrapper = shallowMount(RepoList, { store });
        expect(wrapper).toMatchSnapshot();
    });
});