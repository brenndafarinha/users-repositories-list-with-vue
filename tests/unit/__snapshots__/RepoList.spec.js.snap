// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoList renders correctly 1`] = `
<v-container fluid="">
  <v-row justify="center"></v-row>
  <div class="text-center mt-4">
    <!---->
  </div>
</v-container>
`;
